pkg/controller/cyclenoderequest/transitioner/node.go

```diff
@@ -85,6 +99,14 @@ func (t *CycleNodeRequestTransitioner) addNamedNodesToTerminate(kubeNodes []core
 		for _, kubeNode := range kubeNodes {
 			if kubeNode.Name == namedNode {
 				foundNode = true
+
+				t.cycleNodeRequest.Status.NodesAvailable = append(
+					t.cycleNodeRequest.Status.NodesAvailable,
+					v1.CycleNodeRequestNode{
```

Comment:
There is an update for this object which adds a new `NodeGroupName` field : https://github.com/atlassian-labs/cyclops/blob/master/pkg/apis/atlassian/v1/cyclenoderequest_types.go#L73. 

That field will be used when a node is being attached back during healing process as now we support cycling multiple nodes groups and it needs to know which node group that node will be attached back to. 

That information can be found inside the nodeGroupInstances. similar case can be found here: https://github.com/atlassian-labs/cyclops/blob/master/pkg/controller/cyclenoderequest/transitioner/transitions.go#L145

Reply:
Actually I missed that one below in my previous PR which also needs to be fixed.

Reply:
I've added them